### DISCUSSION TITLE
Is it possible to get 1.53.0 released.

### DISCUSSION THREAD

**Discussion by asadpanda** [golangci/golangci-lint#3806] [Q&A]
There are 3 linters that will be made available in the future release 1.53.0 according to https://golangci-lint.run/usage/linters/

- gosmopolitan
- tagliatelle
- zerologlint

Is it possible to have 1.53.0 released or a date when we should expect it?

**Comment by ldez** [comment]
The v1.53 has been released https://github.com/golangci/golangci-lint/releases
